client-src/default/webpack.config.js

```diff
@@ -18,11 +18,14 @@ module.exports = {
     ],
   },
   plugins: [
-    new webpack.NormalModuleReplacementPlugin(/\/clients\//, (resource) => {
-      resource.request = resource.request.replace(
-        /\/clients\//,
-        '/../clients/'
-      );
-    }),
+    new webpack.NormalModuleReplacementPlugin(
+      /^\.\/clients\/SockJSClient$/,
```

Comment:
We need search way how we can check what module was loaded from our `client-src` - it is right way don't break other code

Reply:
You mean to test that it is loading the correct module in the webpack compilation? Could we run this webpack config in a test, look at the webpack CLI output, and see if it contains the right path?

Reply:
Yes, but it can solved using `NormalModuleReplacementPlugin` as you use, just need more checks (need check context contains 'webpack-dev-server/client-src' and rewrite only in this case)

Reply:
@evilebottnawi I did something like this, except instead I matched `'/client-src/default'`, because it is possible the dev server is not in a `webpack-dev-server` directory if someone clones it into a different named directory. If we want to be very strict about it, we could match against `path.resolve` to get the exact context we want, but I'm afraid that could result in some problem.

Also, I included backwards slash (`\\`) in my matching string because windows seems to use that for the context paths